serverless/aws/examples/src/main/scala/sttp/tapir/serverless/aws/examples/TerraformConfigExample.scala

```diff
@@ -29,7 +29,9 @@ object TerraformConfigExample extends App {
     memorySize = 1024
   )
 
-  val apiGateway: AwsTerraformApiGateway = AwsTerraformInterpreter.toTerraformConfig(helloEndpoint)
+  val apiGateway: AwsTerraformApiGateway = AwsTerraformInterpreter(terraformOptions).toTerraformConfig(helloEndpoint)
+
+  implicit val encoder: Encoder[AwsTerraformApiGateway] = encoderAwsTerraformApiGateway(terraformOptions)
```

Comment:
we need an encoder now? :)

Reply:
yeah because options are no longer implicit and encoder requires them as implicit parameter.
I can change encoder to explicit def with explicit parameters what do you think ?

Reply:
hm and where do we use the encoder? Maybe at the point of use, we can import the encoder as an implicit value

Reply:
From what I see encoder is required by asJson method from AwsTerraformApiGatewa class